Cargo.toml

```diff
@@ -48,7 +48,7 @@ reqwest = { version = "0.12.15", default-features = false }
 reqwest-middleware = "0.4.2"
 serde = { version = "1.0.219", features = ["derive"] }
 serde-sarif = "0.8.0"
-serde_json = "1.0.140"
+serde_json = { version = "1.0.140", features = ["preserve_order"] }
```

Comment:
Not an issue, but out of curiosity: where do we need order-preserving maps here?

(Specifically I'm curious because this is for `serde_json`, not `serde_yaml`.)

Reply:
Also a leftover from when I was trying the older approach of conversions between JSON and YAML and I needed to have this feature to preserve order of keys in the output document, because otherwise it was sorting it and messing it up entirely. Removed.